sample-apps/segmentation_spleen_scribbles/README.md

```diff
@@ -12,32 +11,24 @@ The flow of this application is outlined below:
 
 In the diagram, the neural network (inference) stage is run only once for a given input volume. The logits for this run are saved and used throughout scribble based updates to the same sample. The inferred label is shown to a user through MONAILabel's client extension. The user provides scribbles to indicate corrections needed, which are transferred to server and used along with the logits and original input volume by a post processing algorithm to improve the label. The process is repeated multiple times and the resulting improved label is saved for future training of the deep learning model.
 
-# Installing pre-requisites
+# Using the app
 
-## 1. Install MONAI with BUILD_MONAI=1
-Uses CRF layer from MONAI, which requires compiling the C++/CUDA code following instructions from [MONAI docs](https://docs.monai.io/en/latest/installation.html#option-1-as-a-part-of-your-system-wide-module). 
-This can be done by uninstalling any previous monai/monai-weekly version and running the following command:
+## Install relevant requirements
 
-`BUILD_MONAI=1 pip install git+https://github.com/Project-MONAI/MONAI#egg=monai`
+`BUILD_MONAI=1 pip install -r requirements.txt`
 
 Another way is use docker for MONAI
 
 `docker run --gpus all --rm -ti --ipc=host --net=host -v /xyz:/workspace projectmonai/monai:latest`
 
+## Running the app
 
-## 2. Install SimpleCRF
-Current application provides an option to switch to SimpleCRF library for doing the CRF/GraphCut part. This can be installed as:
-
-`pip install simplecrf`
-
-# Running the app
-
-## Server
+### Server
 On the server side, run server app using the following command:
 
 `CUDA_VISIBLE_DEVICES=0 ./monailabel start_server --app ../sample-apps/segmentation_spleen_postproc/ --studies /path/to/dataset/Task09_Spleen/imagesTrSmall/`
```

Comment:
can we change it to `monailabel start_server --app .... `

users must have already installed the monailabel from pip

Reply:
Fixed